### PR TITLE
[IMP] base_tier_validation_forward: allow user to select backward

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
As mentioned here: https://github.com/OCA/server-ux/pull/272 I'm proposing a way so the user can be the one deciding if the final decision should be made by another person or just ask someone to review it.

Let me know what you think.

### CODE REVIEW

**Comment by kittiu** [comment]
@AdriaGForgeFlow tested and like it. Thanks for your addition.
Can you add your name on contributor list too?

File: base_tier_validation_forward/models/tier_validation.py [hunk 1]
```diff
@@ -18,6 +19,16 @@ def _compute_can_forward(self):
...
 
+    def _compute_can_backward(self):
+        for rec in self:
+            if not rec.can_review:
+                rec.can_backward = False
+                continue
+            sequences = self._get_sequences_to_approve(self.env.user)
+            reviews = rec.review_ids.filtered(lambda l: l.sequence in sequences)
+            definitions = reviews.mapped("definition_id")
+            rec.can_backward = True in definitions.mapped("backward")
```
Comment (kittiu) [inline base_tier_validation_forward/models/tier_validation.py]: \





```suggestion
    def _compute_can_backward(self):
        for rec in self:
            if not rec.can_review:
                rec.can_backward = False
                continue
            sequences = self._get_sequences_to_approve(self.env.user)
            reviews = rec.review_ids.filtered(lambda l: l.sequence in sequences)
            definitions = reviews.mapped("definition_id")
            rec.can_backward = True in definitions.mapped("backward")
```

Should we merge this into `_compute_can_forward`? They are almost identical. May be we can change the function name to `_compute_forward_backward`?

**Comment by AdriaGForgeFlow** [comment]
> @AdriaGForgeFlow tested and like it. Thanks for your addition. Can you add your name on contributor list too?

Done :+1:

File: base_tier_validation_forward/models/tier_validation.py (hunk 1 above)
Comment (AdriaGForgeFlow) [reply base_tier_validation_forward/models/tier_validation.py]: Yes, I agree and have updated the code

**Pull request merged**